tests/ValueFormatters/FormatterOptionsTest.php

```diff
@@ -105,7 +105,7 @@ public function testSetOption() {
 
 		foreach ( $values as $value ) {
 			$formatterOptions->setOption( $value[0], $value[1] );
-			$this->assertEquals( $value[1], $formatterOptions->getOption( $value[0] ) );
+			$this->assertSame( $value[1], $formatterOptions->getOption( $value[0] ) );
```

Comment:
This compares stuff from the `$values` array above, which is guaranteed to contain nothing but primitives.